### PR TITLE
HOMS-142 Add fileupload component

### PR DESCRIPTION
**Pull request by GideonEXE**

### CODE REVIEW

File: hbw/app/assets/javascripts/hbw/components/form/file_upload.js.jsx.coffee [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+      render: ->
+        cssClass = @props.params.css_class
+        cssClass += ' hidden' if this.hidden
+
+        label = @props.params.label
+        labelCss = @props.params.label_css
+
+        formGroupCss = 'form-group'
+
```
Comment (sugarfree1) [inline hbw/app/assets/javascripts/hbw/components/form/file_upload.js.jsx.coffee]: Extra new line.

File: hbw/app/assets/javascripts/hbw/components/form/file_upload.js.jsx.coffee [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+      mixins: [DeleteIfMixin]
+
+      render: ->
+        cssClass = @props.params.css_class
+        cssClass += ' hidden' if this.hidden
+
+        label = @props.params.label
+        labelCss = @props.params.label_css
+
+        formGroupCss = 'form-group'
```
Comment (sugarfree1) [inline hbw/app/assets/javascripts/hbw/components/form/file_upload.js.jsx.coffee]: Do you really need variable for this?

File: hbw/app/models/hbw/fields/file_upload.rb [hunk 3]
```diff
@@ -0,0 +1,10 @@
+module HBW
+  module Fields
+    class FileUpload < Ordinary
+
```
Comment (sugarfree1) [inline hbw/app/models/hbw/fields/file_upload.rb]: Extra new line.

**Comment by GideonEXE** [comment]
UPD

**Comment by kotlas92** [comment]
Hallelujah!

**Pull request merged**